src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -773,7 +786,7 @@ public static PsiElement getBreakStatementOwner(@NotNull PsiElement breakStateme
                                                            GoSelectStatement.class, GoFunctionLit.class);
     return owner instanceof GoFunctionLit ? null : owner;
   }
-  
```

Comment:
Yarrr! Please review your changes before push and provide a separate commit with cleanup if needed in the same PR.